### ISSUE TITLE
Segfault in docker-compose v1.5.0

### ISSUE THREAD

**Issue by armanzor** [leighmcculloch/vagrant-docker-compose#57]
in version 1.5.0
docker-compose is not been installing properly
in 1.4.0 it was ok

**Comment by leighmcculloch** [comment]
Thanks for reporting. @armanzor Could you share the log output, Vagrantfile, which operating systems and versions for the host and the guest?

**Comment by leighmcculloch** [comment]
@armanzor Are you able to share more details?

**Comment by armanzor** [comment]
Hi @leighmcculloch 
Sorry for the pause
Host operation system: Windows 10
Guest OS: Centos7, Ubuntu Bionic
Vagrantfile example:
```
Vagrant.configure("2") do |config|
  config.vm.box = "centos/7"
  config.vm.provision :docker
  config.vm.provision :docker_compose
end
```
Enter to VM with `vagrant ssh` and try `docker-compose info`

**Comment by leighmcculloch** [comment]
Thanks. Could you share the segfault output?

**Comment by leighmcculloch** [comment]
@armanzor `docker-compose info` is not a valid command. Although I wouldn't expect a segfault. I've tested this with the `centos/7` box and with plugin v1.5.0 I don't see a segfault. I don't have a Windows host to test with though.

Can you share the vagrant output that shows the installation of `docker_compose`? I'm particularly interested in which machine and kernel name is detected. It should be something like the following output, which shows machine x86_64 and kernel Linux.

```
==> default: Running provisioner: docker_compose...
    default: Checking for Docker Compose installation...
    default: Getting machine and kernel name from guest machine...
    default: Downloading Docker Compose 1.24.1 for Linux x86_64
    default: Uploading Docker Compose 1.24.1 to guest machine...
    default: Installing Docker Compose 1.24.1 in guest machine...
    default: Symlinking Docker Compose 1.24.1 in guest machine...
```

Can you also share the output of the segfault?

**Comment by karousn** [comment]
+1

**Comment by leighmcculloch** [comment]
@armanzor @karousn This bug has been fixed in `v1.5.1` which is now released. Please reply here if you continue to see the same issue. You can update to `v1.5.1` by running:

```
vagrant plugin update vagrant-docker-compose
```

**Comment by leighmcculloch** [comment]
For any guest machine that already had `docker-compose` installed in, you'll need to remove the `docker-compose` and `docker-compose-1.24.1` binaries from `/usr/local/bin/`. You can then run `vagrant provision` to rerun the install of docker-compose.
